recipe/meta.yaml

```diff
@@ -11,18 +11,22 @@ source:
   sha256: a777e07633d83d546c55706420179551c8e01075b53c497dcf8ae4036766bc66
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   build:
     - python
     - setuptools
     - cython
-    - numpy  x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
   run:
     - python
-    - numpy  x.x
+    - numpy >=1.7  # [py27]
+    - numpy >=1.9  # [py35]
+    - numpy >=1.11  # [py36]
```

Comment:
Building against a "newish" numpy does not create a package which works with older numpy versions. It is necessary to specify the run requirement numpy be strictly greater than or equal to the build version.  For example, installing the pandas from `defaults` built against numpy 1.12 and then force installing numpy 1.9 results in the following import time error:

```
$ python -c "import pandas"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/jhelmus/anaconda/envs/test/lib/python2.7/site-packages/pandas/__init__.py", line 26, in <module>
    from pandas._libs import (hashtable as _hashtable,
  File "/Users/jhelmus/anaconda/envs/test/lib/python2.7/site-packages/pandas/_libs/__init__.py", line 3, in <module>
    from .tslib import iNaT, NaT, Timestamp, Timedelta, OutOfBoundsDatetime
  File "pandas/_libs/src/numpy.pxd", line 157, in init pandas._libs.tslib (pandas/_libs/tslib.c:117317)
ValueError: numpy.dtype has the wrong size, try recompiling. Expected 88, got 96
```  

Although conda-forge does not provide numpy version less than 1.10 they are available in `defaults` and I believe should be supported.  I am willing to submit PRs to get `conda-forge` version of these older numpy version if that is desired.

As for the BLAS issue, any packages which links directly to BLAS/LAPACK should be pinning their BLAS implementation and versions.  I suspect there are packages in the channel which do not do this correctly, these should be updated and the un-pinned packages removed.  This BLAS pinning limits the the supported versions of numpy available to the package via the solver.  I do not think this limitation should be introduced by overloading the numpy package to include a BLAS version.